.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+    "singleQuote": true,
+    "useTabs": true,
+    "tabWidth": 4,
```

Comment:
oops, let's remove this, o-things use eslint :)

Reply:
Sure